src/components/Header/Header.js

```diff
@@ -9,15 +9,36 @@ export default class Header extends Component {
 
     render() {
         return (
-            <header>
-                <div className="header-container">
-                    <div className="logo-container">RocketCare</div>
-                    <div>
-                        <Menu/>
+            <header className="Header">
+                <Link to="/" className="Header__logo">RocketCare</Link>
+                <div className="Header__links">
+                    { this.props.user &&
+                    <>
```

Comment:
Can `<>` here and `</>` on line 34 be removed?

Reply:
Those empty tags are React fragments that allow us to "group a list of children without adding extra nodes to the DOM". 

https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html